src/cli/README.md

```diff
@@ -355,12 +355,71 @@ Done
 
 Print border agent state.
 
+Possible states are
+
+- `Stopped` : Border Agent is stopped
+- `Started` : Border Agent is running with no active connection with external commissioner.
+- `Active` : Border Agent is running and is connected with an external commissioner.
+
 ```bash
 > ba state
 Started
 Done
 ```
 
+### ba ephemeralkey
+
+Indicates whether or not an ephemeral key is in-use by Border Agent.
+
+Requires `OPENTHREAD_CONFIG_BORDER_AGENT_EPHEMERAL_KEY_ENABLE`.
+
+```bash
+> ba ephemeralkey
+not in-use
+Done
+
+> ba ephemeralkey set a10b20c3d151000e60d16 1000
+Done
+
+> ba ephemeralkey
+in-use
+Done
+```
+
+### ba ephemeralkey set \<passphrase\> \[timeout\] \[port\]
+
+Sets ephemeral key for a given timeout duration.
+
+Requires `OPENTHREAD_CONFIG_BORDER_AGENT_EPHEMERAL_KEY_ENABLE`.
+
+The ephemeral key can be set when Border Agent is already running and is not currently connected to any external commissioner (i.e., `ba state` gives `Started`).
+
+The `passphrase` string is directly used as the ephemeral PSK (excluding the trailing null `\0` character). Its length MUST be between 6 and 32, inclusive.
```

Comment:
Good point. I will change the example string to make it clear it can be any string.  

The sequence of numbers I used here is a kind of fun easter egg. The sequence 10, 20, 3, 15, 1000, 60, 16 is from a children song in Persian/Farsi (it rhymes nicely in Farsi). I added letters in between to indicate the phrase can be a string but did not notice that it makes it look like a hexadecimal number. 😄 

We can include spaces in all CLI input by escaping them with the `\` character. More details are documented at the beginning of the `README.md`file.

Reply:
Updated in new push.

Reply:
thanks, resolved